python/gen3authz/client/arborist/client.py

```diff
@@ -130,7 +211,9 @@ def healthy(self):
             bool: whether arborist service is available
         """
         try:
-            response = ArboristResponse(requests.get(self._health_url), expect_json=False)
+            response = self.get(
+                self._health_url, retry=False, timeout=1, expect_json=False
```

Comment:
This is the only time the client code should specify a timeout right? (Because it will override user-defined default timeout given in arb client init?)